### DISCUSSION TITLE
how to add a Print icon on the top right corner of the Github page.

### DISCUSSION THREAD

**Discussion by lowrence-k1** [community/community#112563] [Discussions]
### Select Topic Area

Question

### Body

Hi ,
how to add a Print icon on the top right corner of the Github page.
I use HUGO for UI
and Visual Studio Code for writing the code.

This print icon should be available Globally (that means, available on all the pages). User just has to click on the icon and be able to print that specific page.

Thanks. Waiting for a quick reply please...

**Comment by queenofcorgis** [comment]
Hi @lowrence-k1 ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have closed this duplicate discussion, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

Please note: the Discussions category is solely related to conversations around the GitHub product Discussions.
